ddtrace/appsec/_iast/_taint_tracking/Initializer/Initializer.cpp

```diff
@@ -1,10 +1,13 @@
 #include "Initializer.h"
 
+#include <mutex>
 #include <thread>
 
 using namespace std;
 using namespace pybind11::literals;
 
+std::mutex tx_map_mutex;
```

Comment:
This would be better as a member of Initalizer I think. Also, should the name be active_map_addreses_mutex? The tx_map is ThreadLocal so there is no sharing with it.